include/CppUTest/UtestMacros.h

```diff
@@ -200,6 +200,29 @@
 #define UNSIGNED_LONGS_EQUAL_LOCATION(expected, actual, text, file, line)\
   { UtestShell::getCurrent()->assertUnsignedLongsEqual((unsigned long)expected, (unsigned long)actual, text, file, line); }
 
+#ifdef CPPUTEST_USE_LONG_LONG
```

Comment:
> Anyways, I still suggest to remove the #ifdef even though the error is much less clear. Reason is that not defining USE_LONG_LONG is quite likely, whereas calling the long long assert on a system that doesn't support long long is rather unlikely. So, to get the best default, the #ifdef can be removed.

I'm not quite sure what you mean by this.  Defining the macros in cases where they can't work seems misleading (I conceded on this point for the virtual calls, since there can be situations where leaving them undefined causes trouble, but that is not the case here).

And if someone does not define `USE_LONG_LONG` and does not use the long long asserts, then there's obviously no errors.

So really, which error message is "better" seems like the most important consideration.